File: code-coaches/easy/fruit-bowl/FruitBowl.swift
func getApplePies(fruitCount: Int) -> Int {
    let apples = fruitCount / 2
	return Int(apples / 3)
}

func main() {
    let fruitCount = Int(readLine()!)!
	print(getApplePies(fruitCount: fruitCount))
}

main()

File: code-coaches/easy/fruit-bowl/FruitBowl.cs
using System;

class FruitBowl
{
    static int GetApplePies(int fruitCount)
    {
        int appleCount = fruitCount / 2;
        return appleCount / 3;
    }

    static void Main(string[] args)
    {
        int fruitCount = Convert.ToInt32(Console.ReadLine());
        Console.WriteLine(GetApplePies(fruitCount));
    }
}